specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -435,15 +435,15 @@
               },
               {
                 "value": "OverQuota",
-                "description": "Reached daily cap quota, ingestion stopped."
+                "description": "Stopped ingestion following daily cap quota reach."
               },
               {
                 "value": "SubscriptionSuspended",
-                "description": "Ingestion stopped following suspended subscription."
+                "description": "Stopped ingestion following subscription getting into suspended state."
```

Comment:
The subscription is suspended and ingestion was stopped.

Reply:
"Stop data ingestion following suspended subscription"